app/components/autocomplete/date_suggestion/date_suggestion.js

```diff
@@ -158,6 +158,7 @@ export default class DateSuggestion extends PureComponent {
             >
                 {Boolean(calendarWidth) &&
                 <CalendarList
+                    testID={'autocomplete.date_suggestion'}
```

Comment:
Please remove enclosing parentheses where unnecessary. Please check other testIDs below. Thanks.
```suggestion
                    testID='autocomplete.date_suggestion'
```